### ISSUE TITLE
Comments formating style and vector sign 

### ISSUE THREAD

**Issue by Jordanesku** [orhunulusahin/praatvscode#2]
First of all, thank you for your effort. Here's what I noticed: comments and everything after vector sign (for ex. var# = {}) is styled lika a comment.

For example:
`### this is a comment ###`
`fileList$# = fileNames$#("*.wav")`

On second line everything after # is in italic and gray font like it's a comment

**Comment by orhunulusahin** [comment]
Thank you for the feedback!

The regex for comments was greedy and captured anything after a '#' or ';'. It now only captures these characters if they're preceded by a space, token boundary, or line start. Since '$' is not considered a token boundary in Praat, vectors should no longer break comments from 0.11.0 onward.

**Comment by Jordanesku** [comment]
Sorry I didn't mentioned that earlier, but a$# = {} is a string
vector, but there are also numeric vectors like something# =
{1,2,3,4}. And things get complicated even further: someMatrix## = {{
3, 6, 2 }, { 8, 2, 1 }} - this is matrix example. Yea, In Praat Script
Editor everything works because you can put comments only in a new
line. And if only it could be that char combination is limited to
#={, but you can have something in loop like
something# [i] =  i + i

or
squares# = zero# (100,10000)
x =  numberOfRows (a##)

etc. I think maybe it's too many variations for regex match, but I
don't have enough experience to claim that. Thanks for the effort
anyway adn doing a grat job!!!
Best regards!
Jordan

**Comment by orhunulusahin** [comment]
Right, I had completely overlooked the variation in vectors. I've re-opened the issue, but I'm confident all vector use cases can eventually be captured correctly through detailed enough regex configs. I'll return to this in future updates.
